app/planets_routes.py

```diff
@@ -0,0 +1,138 @@
+from app import db
+from app.models.planet import Planet
+from app.models.moon import Moon
+from flask import Blueprint, jsonify, abort, make_response, request
+from sqlalchemy import desc, asc
+
+
+# ---------helper functions--------
+def sort_planet_query(planet_query, sort_query, order_query):
+    if not sort_query:
+        sort_query = "name"
+
+    if not order_query:
+        order_query = "asc"
+
+    if order_query == "desc":
+        planet_query = planet_query.order_by(desc(sort_query))
+    else:
+        planet_query = planet_query.order_by(asc(sort_query))
+
+    return planet_query
+
+
+
```

Comment:
Tiny best practices feedback - we typically only want a single blank line to separate functions in a file. If we have blocks of functions that do different things, then we might consider using 2 blank lines. A solid example of this are lines 37 and 38 where we have 2 blank lines separating the block of helper functions from the route functions that follow.